README.md

```diff
@@ -53,3 +54,11 @@ Feel free to raise and fix issues.
 For any questions, join #oshc-dev on Slack. Get an invite [here](https://opensourcehelp.herokuapp.com/).
 
 Note : All design related tasks have reward associated with them.
+
+
+<!-- develop -->
+
+[travis-image]: https://img.shields.io/travis/dbader/node-datadog-metrics/master.svg?style=flat-square
```

Comment:
@Logan1x You should use  [this link](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io.svg?branch=develop). We want the Travis-CI built status for `our` repo and not `node-datadog-metrics` repository.